docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -1,44 +1,107 @@
+Using asset checks, you can define and execute data quality checks on your [data assets](/concepts/assets/software-defined-assets) directly in Dagster. Each asset check tests some property of a data asset, such as:
 
-When viewing an asset in Dagster’s UI, you can see all of its checks, and whether they’ve passed, failed, or haven’t run.
+- Ensuring a particular column, like an `ID`, doesn't contain null values
+- Verifying that a tabular asset adheres to a specified schema
+- If the asset's data is in need of a refresh
+
+Assets, their checks, and the results of those checks can be viewed in the Dagster UI, providing you with a unified view of your pipeline's health. For example:
+
+<!-- ![Status of checks for an asset in the Dagster UI](/images/concepts/assets/asset-checks/asset-details-asset-checks-tab.png) -->
+
+<Image
+alt="Status of checks for an asset in the Dagster UI"
+src="/images/concepts/assets/asset-checks/asset-details-asset-checks-tab.png"
+width={1500}
+height={802}
+/>
 
 ---
 
 ## Benefits
 
 Using asset checks helps you:
 
-- Ensure that data produced by your pipeline is timely and high quality
-- Catch problems in the source data that your pipeline depends on
+- Create a single place in the Dagstser UI to learn about and investigate issues in your data pipeline
+- Ensure that data produced by your pipeline is timely and high quality by applying basic suites of tests to all assets
+- Identify problems in the source data that your pipeline depends on, such as checking data quality on source assets where dbt tests aren't possible
 - Communicate what the data produced by your pipeline is expected to look like (aka act as a data contract)
-- Block downstream assets from materializing when issues arise
+- Raise warnings or block downstream assets from materializing when issues arise
+- Track asset check results in the UI, allowing you to monitor data quality metrics over time
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Assets](/concepts/assets/software-defined-assets)
 
 ---
 
 ## How it works
 
-Asset checks are [defined in code](/concepts/assets/asset-checks/define-execute-asset-checks) alongside the asset(s) they target. You can use Python to define them, or if using dbt, load your [dbt tests into Dagster as asset checks](/integrations/dbt/reference#loading-dbt-tests-as-asset-checks). Refer to the [Defining and executing asset checks](/concepts/assets/asset-checks/define-execute-asset-checks) guide for more info.
+An asset check is a data quality check, [defined in code](/concepts/assets/asset-checks/define-execute-asset-checks), that targets one or more Dagster assets. You can use Python to define them, or if using dbt, load your [dbt tests into Dagster as asset checks](/integrations/dbt/reference#loading-dbt-tests-as-asset-checks).
```

Comment:
While most asset checks are data quality checks, some are not. E.g. freshness checks don't really qualify as data quality checks. Someone could also write an asset check to detect private data making it to places that it's not supposed to.

Reply:
That's fair! I'll update this to be a bit more broad.